music21/musicxml/xmlToM21.py

```diff
@@ -6157,25 +6096,24 @@ def testStaffLayout(self):
                             format='musicxml',
                             # forceSource=True
                             )
-        #c = corpus.parse('demos/layoutTest.xml')
+        # c = corpus.parse('demos/layoutTest.xml')
         layouts = c.flat.getElementsByClass('LayoutBase').stream()
         systemLayouts = layouts.getElementsByClass('SystemLayout')
         self.assertEqual(len(systemLayouts), 42)
         staffLayouts = layouts.getElementsByClass('StaffLayout')
-#         for i,p in enumerate(c.parts):
-#             print(i)
-#             for l in p.flat.getElementsByClass('StaffLayout'):
-#                 print(l.distance)
+        #         for i,p in enumerate(c.parts):
```

Comment:
Comments such as these were designed so that the comment could be deleted and the code would run.  Now if the comment is deleted we get an indentation error.